.goreleaser.yml

```diff
@@ -1,7 +1,7 @@
 project_name: kubeone
 builds:
 - ldflags:
-  - "-s -w -X github.com/kubermatic/kubeone/pkg/cmd.version={{.Version}} -X github.com/kubermatic/kubeone/pkg/cmd.commit={{.FullCommit}} -X github.com/kubermatic/kubeone/pkg/cmd.date={{.Date}}"
+  - "-s -w -extldflags=-zrelro -extldflags=-znow -X k8c.io/kubeone/pkg/cmd.version={{.Version}} -X k8c.io/kubeone/pkg/cmd.commit={{.FullCommit}} -X k8c.io/kubeone/pkg/cmd.date={{.Date}}"
```

Comment:
I have updated the ldflags to use the same flags as in the Makefile. Were there any reason why we didn't have `-extldflags=-zrelro -extldflags=-znow` flags in the GoReleaser config before? We have them in the Makefile, but not here.